docs/Changelog.md

```diff
@@ -10370,6 +10370,49 @@ This version of the operator has been available since version 11 of the default
 <dd>Constrain input and output types to float tensors.</dd>
 </dl>
 
+### <a name="ConcatFromSequence-11"></a>**ConcatFromSequence-11**</a>
+
+  Concatenate a sequence of tensors into a single tensor.
```

Comment:
Nit: Theoretically it is possible to have a `Sequence` of other types, such as `Map` or `Sequence`. It may be helpful to add a simple clarification that this op is intended for `Sequence` of `Tensor` only (it is implied from the phrasing, but I suggest being explicit).

Reply:
Yes we need to be explicit on types. The input type constraint shows `sequence<tensor<elem_type>>`, that should be enough ..